### DISCUSSION TITLE
iOS widget support

### DISCUSSION THREAD

**Discussion by jpike88** [community/community#3113] [Mobile]
Would like widgets for:

- open PRs against a given project/a
- open issues
- branches listed by last updated

**Comment by eliperkins** [comment]
@jpike88 these are all great ideas, and widgets are very much on our minds as well! Stay tuned. 😎

> branches listed by last updated

This one is interesting to me. What's your use-case for using a widget to see branches?

**Reply by jpike88** [reply]
A branch may be updated but no PR opened yet, it's just a useful way to gauge activity

**Comment by AustinConlon** [comment]
Another idea is a widget that's a view into your inbox. The contribution graph widget is a start, but I'd like to see more done here.
